Marlin/Marlin_main.cpp

```diff
@@ -1770,6 +1778,26 @@ static void homeaxis(AxisEnum axis) {
       }
     #endif
 
+    #ifdef BABYSTEPPING
+      if(axis == Z_AXIS)
+      {
+        baby_max_endstop[axis] = Z_BABY_DEFAULT_MAX_POS;
+        baby_min_endstop[axis] = Z_BABY_DEFAULT_MIN_POS;
+      }
+      #ifdef BABYSTEP_XY
+        else if(axis == X_AXIS)
+        {
+          baby_max_endstop[axis] = X_BABY_DEFAULT_MAX_POS;
+          baby_min_endstop[axis] = X_BABY_DEFAULT_MIN_POS;
+        }
+        else if(axis == Y_AXIS)
+        {
+          baby_max_endstop[axis] = Y_BABY_DEFAULT_MAX_POS;
+          baby_min_endstop[axis] = Y_BABY_DEFAULT_MIN_POS;
+        }
+      #endif //BABYSTEP_XY
+    #endif //BABYSTEPPING
```

Comment:
What else is there? (parentheses)  Don't misinterpret my comment. Generally, it does very well in converting from one style to another. However there are occasionally things that don't come out quite as expected.